src/utils/upload-file-browser.js

```diff
@@ -4,10 +4,20 @@ const { jsonApiError } = require('./simple-fetch')
 const { isFunction } = require('./typecheck')
 
 const uploadFileBrowser = (options = {}) => {
+  const metadataHeader = 'x-amz-meta-'
+  const versions = options.versions
```

Comment:
Replace `versions` with `variations` in the whole file.

One question for the metadata-header. I think we should add an additional prefix to know, that these metadata is about image converting.

Perhaps `x-amz-meta-ic-` (image converter) or `x-amz-meta-variation-` or `x-amz-meta-image-` or what you think best.